committee-steering/sig-governance.md

```diff
@@ -0,0 +1,629 @@
+- The role of project leadership
+- The process for making technical decisions
+- The process for ensuring continued test health
+
+## How to use this template
+
+SIGs are diverse in nature, and may have different goals based
+on what areas of the project the SIG touches.  Accordingly SIGs will
+need to adapt this template to their specific needs.  This may mean
+removing or changing roles, responsibilities, and processes.  When
+diverging from this template, please help us understand why in the
+PR description so we can improve the process and iterate on this
+document.
+
+**Note:** There are 2 charter template versions:
+
+- a super-lite TL;DR version that only provides a abstract structure.
+- a detailed version that attempts to concretely define a high
+  degree of structure.
+
+SIGs may either choose to start with the TL;DR charter and add
+structure from the detailed version, or to start with the detailed
+version and remove structure that does not provide direct value.
+
+**Note:** Specific responsibilities towards the project are required
+for some SIGs and should be included in the charter - e.g. ensuring
+test health.
+
+FAQ:
+
+- [Why is it so long?](#why-is-it-so-long)
+- [Why so much structure?](#why-so-much-structure)
+- [Why use mailing lists for tracking roles?](#why-use-mailing-lists-for-tracking-roles)
+- [Why track membership as a special role?](#why-track-membership-as-a-special-role)
+- [How do the results from the survey fit in?](#how-do-the-results-from-the-survey-fit-in)
+
+---
+
+# SIG ~~Foo~~ Governance TL;DR Version
+
+## Roles
+
+- Project Leads
+  - Run operations and processes governing the SIG
+  - Outgoing Project Leads may propose new ones -
+  - Number: 2-3
+- Technical Leads
+  - Escalation for technical discussions
+  - Outgoing Technical Leads may propose new ones -
+  - Number: 2-7
+- Members
+  - Maintain health of the project
+  - Build new functionality
+
+## Processes
+
+- Agreeing upon decisions
+  - Send for lazy consensus - send to the googlegroup with RFC or KEP in
+    title
+  - Send LastCall email before accepting consensus
+  - Once accepted, decision is commitment requiring alternative proposal
+    to reverse
+- Test health
+  - Setup on call rotation (dashboard and tooling coming soon)
+  - Setup SLO for test health
+
+---
+
+# SIG ~~Foo~~ Governance Long Version
+
+The following document outlines the governance for
+SIG ~~Foo~~.  Its goals are to define the necessary structure for
+SIG ~~Foo~~ to self govern and to define leadership responsibilities
+within the SIG.
+
+This charter may be updated the members of SIG foo through lazy
+consensus.
+
+This document is broken up into the following parts:
+
+- Roles within the SIG and their responsibilities
+- Processes used by members of the SIG
+
+## Roles and responsibilities
+
+This section describes the various roles governing the SIG.
+
+### Project leads
+
+Canonical list of project leads: kubernetes-sig-~~foo~~-project-leads@googlegroups.com
```

Comment:
OWNERs files aren't sufficient for any sort of collaboration with any sort of group - e.g. email / messaging,  calendar invites,  docs sharing - don't work.  Teams I have worked on typically use another tool for tracking roles within the team or project for this reason.

Perhaps the OWNERs file could be a system of record for the list so that it can be reviewed and the history of changes tracked, but at least some of the responsibilities (e.g. escalation/resolution, planning priorities and milestones) will require a tool that facilitates collaboration with the group .